index.js

```diff
@@ -1,27 +1,29 @@
 const express = require("express");
-const bodyParser = require("body-parser");
 require("dotenv").config();
 const scrapeController = require("./controllers/webscrapeController");
-
-const app = express();
+const { getDateRange } = require("./models/dates");
 const mongoose = require("mongoose");
-const todoRoutes = require("./routes/todo.js");
 
-main()
-  .then((res) => console.log(res))
-  .catch((err) => console.log(err));
+mongoose.connect(process.env.DB);
+console.log("Connected to the database");
+
+const app = express();
 
-async function main() {
-  await mongoose.connect(process.env.DB);
-}
```

Comment:
this change can be kept, why is this removed.
async function main() {
  await mongoose.connect(process.env.DB);
}